indico/core/signals/event/reminder.py

```diff
@@ -0,0 +1,13 @@
+# This file is part of Indico.
+# Copyright (C) 2002 - 2022 CERN
+#
+# Indico is free software; you can redistribute it and/or
+# modify it under the terms of the MIT License; see the
+# LICENSE file for more details.
+
+from indico.core.signals.event import _signals
+
+
+before_reminder_make_email = _signals.signal('before-reminder-make-email', '''
+Executed before reminder email is created. The parameters to create an email are passed.
```

Comment:
"The parameters to create an email are passed." - in pretty much all the other signals we are clear about what params are passed, what the sender is, etc.